src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -47,7 +47,7 @@
           intent="caution"
           icon="warning"
           title={translate('workflows', 'workflow-query-error-state')}
-          >{$workflowError}</Alert
+          style="overflow-wrap: anywhere">{$workflowError}</Alert
```

Comment:
Added this here so there wouldn't be any adverse impacts on other `Alert` components, but wondering if we want to do an audit and potentially just add this to the `Alert` component itself since there could be other similar cases.